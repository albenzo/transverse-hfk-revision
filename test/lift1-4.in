-i 9 -X [3,4,7,9,1,6,8,5,2] -O [1,2,3,5,8,9,4,7,6] -s -n 1
-i 9 -X [3,4,7,9,1,6,8,5,2] -O [1,2,3,5,8,9,4,7,6] -s -n 2
-i 9 -X [3,4,7,9,1,6,8,5,2] -O [1,2,3,5,8,9,4,7,6] -s -n 3
-i 9 -X [3,4,7,9,1,6,8,5,2] -O [1,2,3,5,8,9,4,7,6] -s -n 4
-i 9 -X [3,4,8,9,1,6,7,5,2] -O [1,2,3,5,7,9,4,8,6] -s -n 1
-i 9 -X [3,4,8,9,1,6,7,5,2] -O [1,2,3,5,7,9,4,8,6] -s -n 2
-i 9 -X [3,4,8,9,1,6,7,5,2] -O [1,2,3,5,7,9,4,8,6] -s -n 3
-i 9 -X [3,4,8,9,1,6,7,5,2] -O [1,2,3,5,7,9,4,8,6] -s -n 4
-i 10 -X [4,2,8,9,10,1,5,6,7,3] -O [1,5,3,10,7,8,9,2,4,6] -s -n 1
-i 10 -X [4,2,8,9,10,1,5,6,7,3] -O [1,5,3,10,7,8,9,2,4,6] -s -n 2
-i 10 -X [4,2,8,9,10,1,5,6,7,3] -O [1,5,3,10,7,8,9,2,4,6] -s -n 3
-i 10 -X [4,2,8,9,10,1,5,6,7,3] -O [1,5,3,10,7,8,9,2,4,6] -s -n 4
-i 10 -X [4,5,9,10,1,7,8,6,3,2] -O [1,2,3,6,8,10,5,9,4,7] -s -n 1
-i 10 -X [4,5,9,10,1,7,8,6,3,2] -O [1,2,3,6,8,10,5,9,4,7] -s -n 2
-i 10 -X [4,5,9,10,1,7,8,6,3,2] -O [1,2,3,6,8,10,5,9,4,7] -s -n 3
-i 10 -X [4,5,9,10,1,7,8,6,3,2] -O [1,2,3,6,8,10,5,9,4,7] -s -n 4
-i 10 -X [3,7,8,9,5,6,4,10,1,2] -O [1,10,4,6,8,3,7,2,5,9] -s -n 1
-i 10 -X [3,7,8,9,5,6,4,10,1,2] -O [1,10,4,6,8,3,7,2,5,9] -s -n 2
-i 10 -X [3,7,8,9,5,6,4,10,1,2] -O [1,10,4,6,8,3,7,2,5,9] -s -n 3
-i 10 -X [3,7,8,9,5,6,4,10,1,2] -O [1,10,4,6,8,3,7,2,5,9] -s -n 4
-i 10 -X [3,8,9,5,6,4,10,1,2,7] -O [1,4,6,8,3,7,2,5,9,10] -s -n 1
-i 10 -X [3,8,9,5,6,4,10,1,2,7] -O [1,4,6,8,3,7,2,5,9,10] -s -n 2
-i 10 -X [3,8,9,5,6,4,10,1,2,7] -O [1,4,6,8,3,7,2,5,9,10] -s -n 3
-i 10 -X [3,8,9,5,6,4,10,1,2,7] -O [1,4,6,8,3,7,2,5,9,10] -s -n 4
-i 10 -X [3,9,5,6,4,10,1,2,7,8] -O [1,6,8,3,7,2,5,9,10,4] -s -n 1
-i 10 -X [3,9,5,6,4,10,1,2,7,8] -O [1,6,8,3,7,2,5,9,10,4] -s -n 2
-i 10 -X [3,9,5,6,4,10,1,2,7,8] -O [1,6,8,3,7,2,5,9,10,4] -s -n 3
-i 10 -X [3,9,5,6,4,10,1,2,7,8] -O [1,6,8,3,7,2,5,9,10,4] -s -n 4
-i 10 -X [3,5,6,4,10,1,2,7,8,9] -O [1,8,3,7,2,5,9,10,4,6] -s -n 1
-i 10 -X [3,5,6,4,10,1,2,7,8,9] -O [1,8,3,7,2,5,9,10,4,6] -s -n 2
-i 10 -X [3,5,6,4,10,1,2,7,8,9] -O [1,8,3,7,2,5,9,10,4,6] -s -n 3
-i 10 -X [3,5,6,4,10,1,2,7,8,9] -O [1,8,3,7,2,5,9,10,4,6] -s -n 4
-i 10 -X [3,10,7,8,9,4,5,6,1,2] -O [1,4,9,2,6,7,10,3,8,5] -s -n 1
-i 10 -X [3,10,7,8,9,4,5,6,1,2] -O [1,4,9,2,6,7,10,3,8,5] -s -n 2
-i 10 -X [3,10,7,8,9,4,5,6,1,2] -O [1,4,9,2,6,7,10,3,8,5] -s -n 3
-i 10 -X [3,10,7,8,9,4,5,6,1,2] -O [1,4,9,2,6,7,10,3,8,5] -s -n 4
-i 10 -X [3,10,7,8,9,4,5,6,2,1] -O [1,4,9,2,6,7,10,3,5,8] -s -n 1
-i 10 -X [3,10,7,8,9,4,5,6,2,1] -O [1,4,9,2,6,7,10,3,5,8] -s -n 2
-i 10 -X [3,10,7,8,9,4,5,6,2,1] -O [1,4,9,2,6,7,10,3,5,8] -s -n 3
-i 10 -X [3,10,7,8,9,4,5,6,2,1] -O [1,4,9,2,6,7,10,3,5,8] -s -n 4
-i 10 -X [3,9,8,1,7,4,5,6,10,2] -O [1,2,5,8,10,6,9,3,4,7] -s -n 1
-i 10 -X [3,9,8,1,7,4,5,6,10,2] -O [1,2,5,8,10,6,9,3,4,7] -s -n 2
-i 10 -X [3,9,8,1,7,4,5,6,10,2] -O [1,2,5,8,10,6,9,3,4,7] -s -n 3
-i 10 -X [3,9,8,1,7,4,5,6,10,2] -O [1,2,5,8,10,6,9,3,4,7] -s -n 4
-i 11 -X [4,10,9,1,8,5,6,7,11,2,3] -O [1,2,6,9,11,7,10,3,5,8,4] -s -n 1
-i 11 -X [4,10,9,1,8,5,6,7,11,2,3] -O [1,2,6,9,11,7,10,3,5,8,4] -s -n 2
-i 11 -X [4,10,9,1,8,5,6,7,11,2,3] -O [1,2,6,9,11,7,10,3,5,8,4] -s -n 3
-i 11 -X [4,10,9,1,8,5,6,7,11,2,3] -O [1,2,6,9,11,7,10,3,5,8,4] -s -n 4
-i 11 -X [3,11,8,9,10,4,5,6,7,2,1] -O [1,4,10,2,6,8,11,7,3,5,9] -s -n 1
-i 11 -X [3,11,8,9,10,4,5,6,7,2,1] -O [1,4,10,2,6,8,11,7,3,5,9] -s -n 2
-i 11 -X [3,11,8,9,10,4,5,6,7,2,1] -O [1,4,10,2,6,8,11,7,3,5,9] -s -n 3
-i 11 -X [3,11,8,9,10,4,5,6,7,2,1] -O [1,4,10,2,6,8,11,7,3,5,9] -s -n 4
-i 11 -X [2,10,7,8,9,3,4,5,6,11,1] -O [1,3,9,11,5,7,10,6,2,4,8] -s -n 1
-i 11 -X [2,10,7,8,9,3,4,5,6,11,1] -O [1,3,9,11,5,7,10,6,2,4,8] -s -n 2
-i 11 -X [2,10,7,8,9,3,4,5,6,11,1] -O [1,3,9,11,5,7,10,6,2,4,8] -s -n 3
-i 11 -X [2,10,7,8,9,3,4,5,6,11,1] -O [1,3,9,11,5,7,10,6,2,4,8] -s -n 4
-i 10 -X [5,3,10,1,2,6,7,8,9,4] -O [1,6,2,4,8,10,3,9,5,7] -s -n 1
-i 10 -X [5,3,10,1,2,6,7,8,9,4] -O [1,6,2,4,8,10,3,9,5,7] -s -n 2
-i 10 -X [5,3,10,1,2,6,7,8,9,4] -O [1,6,2,4,8,10,3,9,5,7] -s -n 3
-i 10 -X [5,3,10,1,2,6,7,8,9,4] -O [1,6,2,4,8,10,3,9,5,7] -s -n 4
-i 9 -X [4,8,9,1,6,7,5,2,3] -O [1,2,5,7,9,3,8,4,6] -s -n 1
-i 9 -X [4,8,9,1,6,7,5,2,3] -O [1,2,5,7,9,3,8,4,6] -s -n 2
-i 9 -X [4,8,9,1,6,7,5,2,3] -O [1,2,5,7,9,3,8,4,6] -s -n 3
-i 9 -X [4,8,9,1,6,7,5,2,3] -O [1,2,5,7,9,3,8,4,6] -s -n 4
-i 9 -X [3,8,1,7,4,5,6,9,2] -O [1,2,5,9,6,8,3,4,7] -s -n 1
-i 9 -X [3,8,1,7,4,5,6,9,2] -O [1,2,5,9,6,8,3,4,7] -s -n 2
-i 9 -X [3,8,1,7,4,5,6,9,2] -O [1,2,5,9,6,8,3,4,7] -s -n 3
-i 9 -X [3,8,1,7,4,5,6,9,2] -O [1,2,5,9,6,8,3,4,7] -s -n 4
-i 9 -X [3,8,7,1,4,5,6,9,2] -O [1,2,9,5,6,8,3,4,7] -s -n 1
-i 9 -X [3,8,7,1,4,5,6,9,2] -O [1,2,9,5,6,8,3,4,7] -s -n 2
-i 9 -X [3,8,7,1,4,5,6,9,2] -O [1,2,9,5,6,8,3,4,7] -s -n 3
-i 9 -X [3,8,7,1,4,5,6,9,2] -O [1,2,9,5,6,8,3,4,7] -s -n 4
-i 9 -X [3,8,6,1,4,5,7,9,2] -O [1,2,9,5,7,8,3,4,6] -s -n 1
-i 9 -X [3,8,6,1,4,5,7,9,2] -O [1,2,9,5,7,8,3,4,6] -s -n 2
-i 9 -X [3,8,6,1,4,5,7,9,2] -O [1,2,9,5,7,8,3,4,6] -s -n 3
-i 9 -X [3,8,6,1,4,5,7,9,2] -O [1,2,9,5,7,8,3,4,6] -s -n 4
-i 9 -X [3,8,5,1,4,6,7,9,2] -O [1,2,9,6,7,8,3,4,5] -s -n 1
-i 9 -X [3,8,5,1,4,6,7,9,2] -O [1,2,9,6,7,8,3,4,5] -s -n 2
-i 9 -X [3,8,5,1,4,6,7,9,2] -O [1,2,9,6,7,8,3,4,5] -s -n 3
-i 9 -X [3,8,5,1,4,6,7,9,2] -O [1,2,9,6,7,8,3,4,5] -s -n 4
-i 9 -X [3,8,4,1,5,6,7,9,2] -O [1,2,9,6,7,8,3,5,4] -s -n 1
-i 9 -X [3,8,4,1,5,6,7,9,2] -O [1,2,9,6,7,8,3,5,4] -s -n 2
-i 9 -X [3,8,4,1,5,6,7,9,2] -O [1,2,9,6,7,8,3,5,4] -s -n 3
-i 9 -X [3,8,4,1,5,6,7,9,2] -O [1,2,9,6,7,8,3,5,4] -s -n 4
-i 9 -X [3,8,4,1,5,6,7,2,9] -O [1,2,9,6,7,8,3,4,5] -s -n 1
-i 9 -X [3,8,4,1,5,6,7,2,9] -O [1,2,9,6,7,8,3,4,5] -s -n 2
-i 9 -X [3,8,4,1,5,6,7,2,9] -O [1,2,9,6,7,8,3,4,5] -s -n 3
-i 9 -X [3,8,4,1,5,6,7,2,9] -O [1,2,9,6,7,8,3,4,5] -s -n 4
-i 9 -X [3,9,8,4,1,5,6,7,2] -O [1,5,2,9,6,7,8,3,4] -s -n 1
-i 9 -X [3,9,8,4,1,5,6,7,2] -O [1,5,2,9,6,7,8,3,4] -s -n 2
-i 9 -X [3,9,8,4,1,5,6,7,2] -O [1,5,2,9,6,7,8,3,4] -s -n 3
-i 9 -X [3,9,8,4,1,5,6,7,2] -O [1,5,2,9,6,7,8,3,4] -s -n 4
