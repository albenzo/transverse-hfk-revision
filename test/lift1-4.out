LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
LL is null-homologous
UR is null-homologous
D1[LL] is null-homologous
D1[UR] is null-homologous
theta_2 is null-homologous
theta_3 is null-homologous
theta_4 is null-homologous
